pop-api/examples/fungibles/tests.rs

```diff
@@ -818,13 +843,13 @@ mod utils {
 		PSP22Error::Custom(status_code.to_string())
 	}
 
-	/// This is used to resolve type mismatches between the `AccountId` in the quasi testing
-	/// environment and the contract environment.
+	// This is used to resolve type mismatches between the `AccountId` in the quasi testing
+	// environment and the contract environment.
 	pub(super) fn account_id_from_slice(s: &[u8; 32]) -> AccountId {
 		AccountId::decode(&mut &s[..]).expect("Should be decoded to AccountId")
 	}
 
-	/// Get the last event from pallet contracts.
+	// Get the last event from pallet contracts.
```

Comment:
```suggestion
	/// Get the last event from pallet contracts.
```
Shouldn't this is `///`?